src/components/GPOSPanel/GPOSPanel.jsx

```diff
@@ -67,17 +101,7 @@ class GPOSPanel extends Component {
         <div className='gpos-panel__stats-potential'>
           <Translate content='gpos.side.info.potential'/>
           <div className='gpos-panel__stats--right'>
-            {gposReward && asset
-              ? <FormattedNumber
-                value={ gposReward && asset
-                  ? gposReward / Math.pow(10, asset.get('precision'))
-                  : gposReward
-                }
-                minimumFractionDigits={ 0 }
-                maximumFractionDigits={ asset.get('precision') }
-              />
-              : 0
-            } {symbol}
+            {`${estimatedRakeReward}%`}
```

Comment:
Does this need to be a template literal?

Reply:
no but changing it would just be to: {estimatedRakeReward + '%'} which is only kb smaller than what it is